ui/app/components/clients/charts/vertical-bar-basic.ts

```diff
@@ -27,21 +27,19 @@ interface ChartData {
 }
 
 /**
- * @module ClientsSyncBarChartComponent
- * Renders a bar chart of secret syncs over time.
+ * @module VerticalBarBasic
+ * Renders a vertical bar chart of counts (@yKey) over time.
  *
  * @example
- * ```js
- * <Clients::SyncBarChart
+ <Clients::Charts::VerticalBarBasic
     @chartTitle="Secret Sync client counts"
     @data={{this.model}}
-    @dataKey="secret_syncs"
+    @yKey="secret_syncs"
```

Comment:
Originally I also abstracted `xKey` but it required passing in formatting functions for both the legend and axes. I reverted the change since these graphs are just for month over month data right now, and expanding on them can happen in a future iteration if necessary.